espnet/transform/cmvn.py

```diff
@@ -106,11 +106,11 @@ def __call__(self, x, uttid=None):
             if self.norm_vars:
                 x = np.multiply(x, self.scale[spk])
 
-        else:
-            if self.norm_means:
-                x = np.subtract(x, self.bias[spk])
+        else:            
```

Comment:
Please delete the extra spaces.
```suggestion
        else:
```